policy/README.md

```diff
@@ -78,14 +80,16 @@ See the [Policy Examples](./examples/README.md) for ways Policy can be implement
 
 ### Authorization
 
-The Policy endpoints should be made public. Authorization is not required.
+The Policy endpoints should be made public. Authorization is not required. Agencies may make reasonable accommodations to manage their endpoints, for example, using an API key that has a clear, public way to obtain.
```

Comment:
I think it would make sense to suggest rate limiting and suggesting ways to query to limit bandwidth. 

Example language: "Agencies may make reasonable accommodations to manage their endpoints, for example, using an API key that has a clear, public way to obtain, using rate limiting depending on the source of the request, and by publishing examples for queries that allow the agency to distinguish the audience of the request. "

Reply:
Thanks @gene I made an update to the language incorporating what you said, and adding some more reasons that came up recently.